### PR TITLE
Allow windows build with vscode

### PR DESCRIPTION
**Pull request by Marr11317**
Resolves: no issue in tracker

using msvc build.bat, this allows to build musescore with vscode.

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/Documentation/blob/master/CodeGuidelines.md)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [x] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

**Comment by Marr11317** [comment]
Note that the other option to have a viable vscode config would be to add .vscode to .gitignore, so it that users could manage that themselves these settings themselves. It would however require more manual setup.

File: .vscode/c_cpp_properties.json [hunk 1]
```diff
@@ -28,26 +28,42 @@
...
                 ]
             }
+        },
+        {
+            "name": "Win32",
+            "includePath": [
+                "${workspaceFolder}/**",
+                "C:/Qt/5.12.10/msvc2017_64/bin",
+                "C:/Qt/5.12.10/msvc2017_64/include/**",
+                "C:/Qt/5.12.10/msvc2017_64/include/QtGui"
```
Comment (Jojo-Schmitz) [inline .vscode/c_cpp_properties.json:52]: So we really need to hardcode these values?

Comment (Marr11317) [reply .vscode/c_cpp_properties.json:52]: Maybe not. These are for not for building anyway. I can just remove them from there and let people choose what they include.

Comment (Jojo-Schmitz) [reply .vscode/c_cpp_properties.json:52]: Something like `${QT_INSTALL_PREFIX}/`maybe?

Comment (Marr11317) [reply .vscode/c_cpp_properties.json:52]: only available with cmake-initialized environments. I'll remove these include paths for the time being. Individuals will add them if they wish.

**Pull request merged**